thirdparty/cmdconfig/commands/cmdtree/tree.go

```diff
@@ -24,7 +24,7 @@ const (
 	// Resources packages.
 	TreeStructurePackage TreeStructure = "directory"
 
-	PkgPrefix = "PKG: "
+	PkgPrefix = "Package "
```

Comment:
This should be `PkgNameFormat = "Package %q:"` and describe it with comment what %q holds.